ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -1238,6 +1290,7 @@ const triggerSet: TriggerSet<Data> = {
           de: 'Gespeichert: ${effect}',
           ja: 'あとで: ${effect}',
           cn: '存储: ${effect}',
+          ko: '기억: ${effect}',
```

Comment:
```suggestion
          ko: '저장: ${effect}',
```

I intended to use '저장' instead of '기억' but forgot